staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -48,6 +48,18 @@ GROUPS_WITH_VERSIONS="$5"
 shift 5
 
 go install ./"$(dirname "${0}")"/cmd/{defaulter-gen,conversion-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+
+SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)"
+KUBE_OPENAPI_PKG=$(
+    cd "${SCRIPT_ROOT}"
+    while [ "${PWD}" != "/" ]; do
+        ls -d -1 "${PWD}"/vendor/k8s.io/kube-openapi 2>/dev/null && exit 0
+        cd ..
+    done
+    echo k8s.io/kube-openapi
+)
+go install "${KUBE_OPENAPI_PKG}"/cmd/openapi-gen
```

Comment:
does code-generator have consumers via the published staging repo? what does this addition do for those consumers?

Reply:
let me check. I wonder whether we shouldn't put a wrapper into k8s.io/code-generator to reuse the path magic.

Reply:
done